app/experimenter/nimbus-ui/src/components/PageHome/FilterBar/index.tsx

```diff
@@ -124,7 +128,12 @@ const FilterSelect = <
           value: fieldValue,
           placeholder: "All " + fieldLabel + "s",
           getOptionLabel: (item: OptionTypeBase) =>
-            item[optionLabelName as string],
+            fieldLabel === "Feature"
+              ? item[optionLabelName as string] +
+                " (" +
+                displayConfigLabelOrNotSet(item["application"], applications) +
+                ")"
```

Comment:
Oh try not to use string concatenation like this, you can use a template literal just like a format string in Python:

https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html